draft-ietf-tls-esni.md

```diff
@@ -1012,39 +985,60 @@ server to include any information it requires to process the second ClientHello.
 
 ## Backend Server {#backend-server}
 
-Upon receipt of an "ech_is_inner" extension in a ClientHello, if the backend
-server negotiates TLS 1.3 or higher, then it MUST confirm ECH acceptance to the
-client by computing its ServerHello as described here.
+Upon receipt of an "encrypted_client_hello" extension of type `inner` in a
+ClientHello, if the backend server negotiates TLS 1.3 or higher, then it MUST
+confirm ECH acceptance to the client by computing its ServerHello as described
+here.
 
-The backend server begins by generating a message ServerHelloECHConf, which is
-identical in content to a ServerHello message with the exception that
-ServerHelloECHConf.random is equal to 24 random bytes followed by 8 zero bytes.
-It then computes an 8-byte string
+The backend server embeds in ServerHello.random a string derived from the inner
+handshake. It begins by computing its ServerHello as usual, except the last 8
+bytes of ServerHello.random are set to zero. It then computes the transcript
+hash for ClientHelloInner up to and including the modified ServerHello, as
+described in {{RFC8446}}, Section 4.4.1. Let transcript_ech_conf denote the
```

Comment:
This seems like a more verbose description of the hash than what was there previously. Why the change?

Reply:
Seems like it was to clarify what exactly the transcript hash should consist of (i.e. the zeroes) or else it becomes a recursive definition.

Reply:
I don't see any difference in the text 🤷